packages/editor/src/components/header/index.js

```diff
@@ -82,21 +83,29 @@ function Header( {
 		};
 	}, [] );
 
-	const hasTopToolbar = isLargeViewport && hasFixedToolbar;
-
 	const [ isBlockToolsCollapsed, setIsBlockToolsCollapsed ] =
 		useState( true );
 
+	const hasCenter = isBlockToolsCollapsed && ! isTooNarrowForDocumentBar;
+	const hasBackButton = useHasBackButton();
+
 	// The edit-post-header classname is only kept for backward compatibilty
 	// as some plugins might be relying on its presence.
 	return (
-		<div className="editor-header edit-post-header">
-			<motion.div
-				variants={ backButtonVariations }
-				transition={ { type: 'tween' } }
-			>
-				<BackButton.Slot />
-			</motion.div>
+		<div
+			className={ clsx( 'editor-header edit-post-header', {
+				'has-back-button': hasBackButton,
+				'has-center': hasCenter,
+			} ) }
+		>
+			{ hasBackButton && (
+				<motion.div
+					variants={ backButtonVariations }
+					transition={ { type: 'tween' } }
+				>
+					<BackButton.Slot />
+				</motion.div>
+			) }
```

Comment:
Will there be a case when a "back button" is missing? Seems like a problematic path for fullscreen users.

Cc @youknowriad (since you worked on this unification recently).

Reply:
I don't understand the question. Would you mind clarifying?

Reply:
I think the "back button" is always present. It can be overridden by third-parties though.